packages/plugin-core/src/constants.ts

```diff
@@ -541,12 +542,12 @@ export const DENDRON_COMMANDS: { [key: string]: CommandEntry } = {
   CONFIGURE_SERVICE_CONNECTION: {
     key: "dendron.configureServiceConnection",
     title: `${CMD_PREFIX} Configure Service Connection`,
-    when: DendronContext.PLUGIN_ACTIVE,
+    when: DendronContext.PLUGIN_ACTIVE && DendronContext.ENABLEEXPORTPODV2,
```

Comment:
I know this is super confusing so let me go over this again (I had to look this up AGAIN as well 😅)

from api docs (still confusing)
https://code.visualstudio.com/api/extension-guides/command#enablement-of-commands

from source code (much more clear)
https://github.com/microsoft/vscode/blob/main/src/vs/workbench/api/common/menusExtensionPoint.ts#L546:L549

So if we want to really block people from using this by any means unless they explicitly set the flag to true, we should add the "enablement" property to the "commands" contribution point rather than adding the "when" property to individual user-facing UI (command palette, menu, keybinding, etc)

adding
```
...
  "commands": [
    ...
    {
      "command": "dendron.configureServiceConnection",
      "title": ...,
      "enablement": "dendron.pluginActive && "dendron.enableExportPodV2"
    }
```

will disable this from all user facing UIs (command palette, menus, keybindings). It won't stop people from opening up the debug console within VSCode and manually triggering `vscode.commands.executeCommand("dendron.configureServiceConnection", ...)`, but at that point we should let them know it could be enabled by a flag.

Reply:
when we ship this, @Harshita-mindfire , can you also add documentation to [[Commands|dendron://dendron.docs/pkg.plugin-core.dev.cook#commands]] about how to add a command in preview following the process here? link to this PR as a reference

Reply:
when you add that doc, can you add me and @hikchoi as reviewewrs?

Reply:
https://github.com/dendronhq/dendron-docs/pull/11